terraform/modules/single_instance/ansible.tf

```diff
@@ -0,0 +1,29 @@
+locals {
+  ansible_env={
+    ANSIBLE_REMOTE_PORT = local.ports.ssh
+    ANSIBLE_REMOTE_USER = local.ssh_user
+    ANSIBLE_HOST_KEY_CHECKING = false
+  }
+  ansible_command="timeout 4m ansible-playbook -c ssh -i ${data.openstack_networking_floatingip_v2.public.address},"
```

Comment:
why do you use the public IP instead of the local _vm private one?

Reply:
Because otherwise users using terraform on their local laptop wont be able to connect to the vm